content/projects/gsoc/2021/project-ideas/semantic-release-version.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: gsocprojectidea
+title: "Semantic Versioning plugin for Jenkins"
+goal: "Automatically determine the next semantic version for a repository"
+category: Plugins
+year: 2021
+status: draft
+sig: docs
```

Comment:
Are you sure about it?

Reply:
@oleg-nenashev are you referring to the sig part?

Reply:
I suspect this should be sig: cloud-native since that is the most likely location for your discussions.  I think it would be great to include in the Docs SIG, but I don't think that we want to burden you with conversations in a SIG meeting that you don't typically need to attend.  Since you're already involved in the cloud-native sig, what if the discussions happened there?

Reply:
Cloud Native sig sounds good, I actually copied the cloudevents & tekton client proposals so they seem to both be in the docs sig atm.